configure.ac

```diff
@@ -15,5 +15,19 @@ AC_SEARCH_LIBS([setupterm], [curses], [], [
     )]
   )
 ])
+AC_DEFUN([AC_MALLOC_OPTIONS], [
+  AC_MSG_CHECKING([for malloc hardening options])
+  case "`uname -s`" in
```

Comment:
If you [`AC_CANONICAL_HOST`](https://www.gnu.org/software/autoconf/manual/autoconf-2.62/html_node/Canonicalizing.html) then you can use `$host_os` in here instead.